mrmustard/lab_dev/states/base.py

```diff
@@ -938,10 +941,12 @@ def __rshift__(self, other: CircuitComponent) -> CircuitComponent:
         with those of a ``DM`` or of a ``Ket``. Returns a ``CircuitComponent`` otherwise.
```

Comment:
```suggestion
        with those of a ``DM`` or of a ``Ket``. Returns a ``CircuitComponent`` otherwise.
```
```suggestion
        with those of a ``DM`` or of a ``Ket``. Returns a ``CircuitComponent`` otherwise and a scalar if there are no wires left.
```